recipe/meta.yaml

```diff
@@ -25,7 +24,7 @@ requirements:
     - python
     - parse >=1.6
     - six
-    - enum34
+    - enum34  # [py2k]
```

Comment:
Take a look at our docs (and please make some comments :wink:):

https://github.com/conda-forge/conda-forge.github.io/pull/350/files

Re-rendering there will update the feedstock to include Python 3.6 in the build matrix.